django-stubs/db/models/manager.pyi

```diff
@@ -51,7 +51,7 @@ class BaseManager(Generic[_T]):
     def bulk_create(
         self, objs: Iterable[_T], batch_size: Optional[int] = ..., ignore_conflicts: bool = ...
     ) -> List[_T]: ...
-    def bulk_update(self, objs: Iterable[_T], fields: Sequence[str], batch_size: Optional[int] = ...) -> int: ...
+    def bulk_update(self, objs: Iterable[_T], fields: Iterator[str], batch_size: Optional[int] = ...) -> int: ...
```

Comment:
Why do you use `Iterator` here and `Iterable` below?

Reply:
Corrected it now.